### PR TITLE
Switches Sinopia search to take options param.

### PR DESCRIPTION
**Pull request by justinlittman**
Rather than a large number of individual params.

### CODE REVIEW

File: src/actionCreators/search.js [hunk 1]
```diff
@@ -6,7 +6,9 @@ import { findAuthorityConfig } from 'utilities/authorityConfig'
 
 // eslint-disable-next-line max-params
 export const fetchSinopiaSearchResults = (query, queryFrom, resultsPerPage, sortField, sortOrder) => dispatch => getSearchResults(
-  query, queryFrom, resultsPerPage, sortField, sortOrder,
+  query, {
+    queryFrom, resultsPerPage, sortField, sortOrder,
```
Comment (jmartin-sul) [inline src/actionCreators/search.js]: looks like this is sticking `resultsPerPage` in a field of the same name, but `getSearchResults` expects a field in the `options` hash called `size`?

File: __tests__/components/search/SinopiaSort.test.js [hunk 2]
```diff
@@ -55,6 +55,8 @@ describe('<SinopiaSort />', () => {
     fireEvent.click(getByText('Label, ascending'))
 
     expect(await findByText('Label, ascending', { selector: '.active' })).toBeInTheDocument()
-    expect(server.getSearchResults).toHaveBeenCalledWith('twain', 0, 15, 'label', 'asc')
+    expect(server.getSearchResults).toHaveBeenCalledWith('twain', {
+      queryFrom: 0, resultsPerPage: 15, sortField: 'label', sortOrder: 'asc',
```
Comment (jmartin-sul) [inline __tests__/components/search/SinopiaSort.test.js]: `resultsPerPage` -> `size`?

File: __tests__/components/search/Search.test.js [hunk 3]
```diff
@@ -138,7 +140,9 @@ describe('<Search />', () => {
     fireEvent.keyPress(getByLabelText('Query'), { key: 'Enter', code: 13, charCode: 13 })
 
     // Called once
-    expect(mockGetSearchResults).toBeCalledWith('foo', 0, 10, undefined, undefined)
+    expect(mockGetSearchResults).toBeCalledWith('foo', {
+      queryFrom: 0, resultsPerPage: 10, sortField: undefined, sortOrder: undefined,
```
Comment (jmartin-sul) [inline __tests__/components/search/Search.test.js]: `resultsPerPage` -> `size`?

File: __tests__/components/search/Search.test.js [hunk 4]
```diff
@@ -110,7 +110,9 @@ describe('<Search />', () => {
     fireEvent.click(container.querySelector('button[type="submit"]'))
 
     // Called once
-    expect(mockGetSearchResults).toBeCalledWith('foo', 0, 10, undefined, undefined)
+    expect(mockGetSearchResults).toBeCalledWith('foo', {
+      queryFrom: 0, resultsPerPage: 10, sortField: undefined, sortOrder: undefined,
```
Comment (jmartin-sul) [inline __tests__/components/search/Search.test.js]: `resultsPerPage` -> `size`?

**Comment by justinlittman** [comment]
@jmartin-sul And this is why I love code review. Fixed.

**Pull request merged**